util/Migrator/DbScripts/2022-12-08_00_OrgUserGroupsAndCollections.sql

```diff
@@ -0,0 +1,42 @@
+IF OBJECT_ID('[dbo].[CollectionUser_ReadByOrganizationUserIds]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[CollectionUser_ReadByOrganizationUserIds];
+END
+GO
+
+CREATE PROCEDURE [dbo].[CollectionUser_ReadByOrganizationUserIds]
+    @OrganizationUserIds [dbo].[GuidIdArray] READONLY
+AS
+BEGIN
+    SET NOCOUNT ON
+
+SELECT
+    CU.*
+FROM
+    [dbo].[OrganizationUser] OU
+    INNER JOIN
+    [dbo].[CollectionUser] CU ON OU.[AccessAll] = 0 AND CU.[OrganizationUserId] = [OU].[Id]
+    INNER JOIN
+    @OrganizationUserIds OUI ON OUI.[Id] = OU.[Id]
+END
+GO
+
+IF OBJECT_ID('[dbo].[GroupUser_ReadByOrganizationUserIds]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[GroupUser_ReadByOrganizationUserIds];
+END
+GO
+
+CREATE PROCEDURE [dbo].[GroupUser_ReadByOrganizationUserIds]
+    @OrganizationUserIds [dbo].[GuidIdArray] READONLY
+AS
+BEGIN
+    SET NOCOUNT ON
+
+SELECT
+    GU.*
+FROM
+    [dbo].[GroupUser] GU
+    INNER JOIN
+    @OrganizationUserIds OUI ON OUI.[Id] = GU.[OrganizationUserId]
+END
```

Comment:
Do we need a `GO` here?

Reply:
I don't believe so, it's at the end of the script and there are no other blocks to go to. Although, looking at other migration scripts its about 50/50 if they have a trailing `GO` command or not...